app/forms/steps/capital/other_property_type_form.rb

```diff
@@ -0,0 +1,11 @@
+module Steps
+  module Capital
+    class OtherPropertyTypeForm < PropertyTypeForm
+      private
+
+      def persist!
+        @property = incomplete_property_for_type || crime_application.properties.create!(property_type:)
```

Comment:
Let's follow Tim's lead on this and put this one back to:
`@property = crime_application.properties.create!(property_type:) `
https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/blob/ce818a26d006644f1c6c701112f18e79ce96eff5/app/forms/steps/capital/other_investment_type_form.rb#L7